infrastructure/code/issue_enhancer_test.go

```diff
@@ -264,7 +264,7 @@ func Test_addIssueActions(t *testing.T) {
 
 		issueData, ok := fakeIssues[0].AdditionalData.(snyk.CodeIssueData)
 		require.True(t, ok)
-		assert.False(t, issueData.HasAIFix)
+		assert.True(t, issueData.HasAIFix)
```

Comment:
With this PR change, is it correct to say that this unit test description needs to be updated too?

It currently reads:

    "Does not include AI fixes if issue is ignored"

However, as far as I understand, we'd like to assert that:

    "Includes AI fixes even if issue is ignored"

Is this correct?

Reply:
Not exactly, the AI fixes are still not included in the code actions and code lenses. Which is what this test verifies. The side-effect of setting `HasAIFix` in this code is not great but I didn't want to refactor the code smell right now since we're in a bit of a time crunch to get this into the CLI before the 26th of June

Reply:
Then updating the unit description to something more align to this context also does not work? 

It's just that the title says `Does not include AI fixes if issue is ignored` and the assertion is `assert.True(t, issueData.HasAIFix)`.

A description like:

```
Sets HasAIFix to true but does not include AI fixes in Code Actions or Code Lenses if issue is ignored
```
could maybe work?

> [!NOTE]  
> This is not a change request

Reply:
I just think it's a bit too verbose. I've changed the test name a bit (and for the one above it, since it was just incorrect) but I don't specify HasAIFix in it. Thanks for the suggestion!